messages/en.json

```diff
@@ -0,0 +1,42 @@
+{
+  "LocaleSwitcher": {
+    "label": "Change language",
+    "locale": "{locale, select, de {🇩🇪 Deutsch} en {🇺🇸 English} es {🇪🇸 Español} other {Unbekannt}}"
```

Comment:
🇩🇪 -> 🇨🇳 ?

Lydon has a Chinese-speaking friend who is usually willing to edit Google translate outputs, I have her number if you want me to send this over

Reply:
It might actually be nice to add Chinese in a separate PR, so that we can have a clean reference for the "+1 language diff" to reference later

Reply:
I agree. I was going to focus on functionality in this PR and then add the others in a followup.